blockindex/contractstaking/indexer_test.go

```diff
@@ -68,12 +68,12 @@ func TestContractStakingIndexerLoadCache(t *testing.T) {
 	// create a stake
 	height := uint64(1)
 	startHeight := uint64(1)
-	handler := newContractStakingEventHandler(indexer.cache, height)
+	handler := newContractStakingEventHandler(indexer.cache)
 	activateBucketType(r, handler, 10, 100, height)
 	owner := identityset.Address(0)
 	delegate := identityset.Address(1)
 	stake(r, handler, owner, delegate, 1, 10, 100, height)
-	err = indexer.commit(handler)
+	err = indexer.commit(handler, height)
```

Comment:
add test height, after success commit, height should be updated

Reply:
fixed